CMakeLists.txt

```diff
@@ -104,6 +104,26 @@ endif()
 option(ZSTD_FROM_INTERNET "Download and use libzstd from the Internet" ${ZSTD_FROM_INTERNET_DEFAULT})
 find_package(zstd 1.1.2 REQUIRED)
 
+option(REDIS_STORAGE_BACKEND "Enable redis secondary storage")
+if(REDIS_STORAGE_BACKEND)
+  find_package(Hiredis 0.13.3)
+  if(DEFINED HIREDIS_INCLUDE_DIRS)
```

Comment:
the `Hiredis_FOUND` should be already set by find_package call

Reply:
But the `HiredisConfig.cmake` is utterly broken.

Reply:
I gave up on it, but I find this generally to be the case... All different.

Reply:
Sadly it seems that cmake behaves diffently on Mac, the LIBDIR is gone